Enhancement: Add additional entries to the parser namespace

CognitiveMist opened the pull request:
Add parsing access for the following:
- AbstractOutfit via prefix `OUTFIT_`
- DamageVariance via prefix `DAMAGE_VARIANCE_`
- EquipClothingSetting via prefixes `ECS_` and `EQUIP_CLOTHING_SETTING_`
- FluidFlavour via prefix `FLUID_FLAVOUR_`
- FluidModifier via prefix `FLUID_MODIFIER_`
- NPCGenerationFlag via `NPC_GEN_FLAG_`
- OutfitType via prefix `OUTFIT_TYPE_`
- SlavePermission via prefix `SLAVE_PERMISSION_`
- TattooType via prefix `TATTOO_`

Add aliases for existing prefixes:
- AbstractStatusEffect via prefix `STATUS_EFFECT_` (alias for `SE_`)
- CoverableArea via prefix `COVERABLE_AREA_` (alias for `CA_`)
- InventorySlot via prefix `INV_SLOT_` (alias for `IS_`)

I have seen most of these come up in conversations in #mod-discussion and/or #developers over the past several months.

Feel free to adjust or remove as needed.

Tested on: 0.4.2
Discord: Phlarx#1765

CognitiveMist committed:
Add additional entries to the parser namespace

Add parsing access for the following:
- TattooType via prefix TATTOO_
- AbstractOutfit via prefix OUTFIT_
- OutfitType via prefix OUTFIT_TYPE_
- EquipClothingSetting via prefix ECS_
- NPCGenerationFlag via NPC_GEN_FLAG_
- SlavePermission via prefix SLAVE_PERMISSION_

CognitiveMist committed:
More parser values:

- FluidModifier via prefix FLUID_MODIFIER_
- FluidFlavour via prefix FLUID_FLAVOUR_

Anonymous-BCFED commented:
`ECS_` seems a bit ambiguous to me.  Maybe `EQUIP_SETTING_` or `CLOTHING_SETTING_`?

CognitiveMist committed:
More parser values:

- DamageVariance via prefix DAMAGE_VARIANCE_

More aliases:

- EquipClothingSetting via prefix EQUIP_CLOTHING_SETTING_
- CoverableArea via prefix COVERABLE_AREA_
- InventorySlot via prefix INV_SLOT_
- AbstractStatusEffect via prefix STATUS_EFFECT_

Innoxia commented:
Thank you for this!
